src/datasources/asset-calibration/types.ts

```diff
@@ -137,3 +138,4 @@ export const AssetTypeOptions = [
   { label: 'System', value: AssetType.SYSTEM },
 ];
 
+export type ExpressionTransformFunction =  (value: string, operation: string, options?: TTLCache<string, unknown>) => string;
```

Comment:
We should add a description to this to help the user understand how and when to use this.